modules/apps/batch-engine/batch-engine-service/src/main/java/com/liferay/batch/engine/internal/reader/BatchEngineImportTaskItemReaderUtil.java

```diff
@@ -61,11 +63,17 @@ public static <T> T convertValue(
 			}
 
 			if (field != null) {
+				JsonDeserialize[] jsonDeserializes = field.getAnnotationsByType(
```

Comment:
This attribute could be calculated inside the `_getObjectMapper` method. You don't use it here

Reply:
https://github.com/liferay-headless/liferay-portal/pull/1580/commits/b6d1f210d3a9fca5948de8448b499d76902ff2d6.